vagen/envs/navigation/benchmark.py

```
@@ -0,0 +1,284 @@
+    LOGGER.info(
+        f"Benchmark: {num_rounds} rounds × {num_clients} clients × {num_steps} steps"
+    )
+    LOGGER.info(f"Server: {base_url}")
+
+    # Warm-up: quick health check
+    import httpx
+    async with httpx.AsyncClient(timeout=10) as hc:
+        try:
+            resp = await hc.get(f"{base_url}/health")
+            LOGGER.info(f"Health check: {resp.json()}")
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)